pkg/controller/machine/controller.go

```diff
@@ -167,6 +173,34 @@ func (r *ReconcileMachine) Reconcile(ctx context.Context, request reconcile.Requ
 	// This must be a copy otherwise the referenced slice will be modified by later machine conditions changes.
 	originalConditions := conditions.DeepCopyConditions(m.Status.Conditions)
 
+	// Check Status.AuthoritativeAPI
+	// If not MachineAPI. Set the paused condition true and return early.
+	if m.Status.AuthoritativeAPI != machinev1.MachineAuthorityMachineAPI {
+		conditions.Set(m, conditions.TrueConditionWithReason(
+			PausedCondition,
+			PausedConditionReason,
+			"The AuthoritativeAPI is set to %s", m.Status.AuthoritativeAPI,
+		))
+		if patchErr := r.updateStatus(ctx, m, ptr.Deref(m.Status.Phase, ""), nil, originalConditions); patchErr != nil {
+			klog.Errorf("%v: error patching status: %v", machineName, patchErr)
+		}
+
+		klog.Infof("%v: setting paused to true and returning early", machineName)
+		return reconcile.Result{}, nil
+	}
+
+	// Set the paused condition to false, continue reconciliation
+	conditions.Set(m, conditions.FalseCondition(
+		PausedCondition,
+		NotPausedConditionReason,
+		machinev1.ConditionSeverityInfo,
+		"The AuthoritativeAPI is set to %s", m.Status.AuthoritativeAPI,
+	))
+	if patchErr := r.updateStatus(ctx, m, ptr.Deref(m.Status.Phase, ""), nil, originalConditions); patchErr != nil {
+		klog.Errorf("%v: error patching status: %v", machineName, patchErr)
+	}
```

Comment:
Is there an update status later in the reconcile? Or one that is deferred? Wondering if this could increases our 409 count if we add additional updates mid flow.

Would ideally expect at most one status update per reconcile

Reply:
- See if we can use existing updates to status rather than introducing a new one.

Reply:
check every path where we exit the reconcile loop (after where we call updatestatus) we dont need the additional one when setting false

Reply:
I don't think this will work without some rejigging - there are a number of places where we exit without calling updateStatus. 

We could just return after the inital call to set paused=false, but that will mean a requeue.